docs/UserGuide.md

```diff
@@ -12,30 +12,54 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+### Installation
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the `.jar` file of the latest release, from [here](https://github.com/AY2425S1-CS2103T-F08-2/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+2. Move the `.jar` file into an empty folder.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+3. Open a command window in that folder.
+3.1 Right-click on the folder.
+3.2 Select `Open in Terminal`.
```

Comment:
Do we want to include step for MacOS?
Right-click -> Services -> New Terminal at Folder

Reply:
OK, will include in next commit push!